README.md

```diff
@@ -1,3 +1,45 @@
-# reqore
+<div align="center">
+  <br><br><br>
+  <img src="./public/logo.png" alt="Unstated Logo" width="500">
+  <br><br><br>
+</div>
 
-ReQore is a UI library for Qorus connected apps.
+# @qoretechnologies/reqore
+
+Reqore contains foundation React UI components for shipping apps that utilize Qore Technologies backend API
```

Comment:
That's not correct - reqore is just a UI library that can be used anywhere (as long as it's React)